doc/tasks/README.md

```diff
@@ -0,0 +1,550 @@
+
+### Multiple-Choice
+
+Die Aufgabenstellung besteht aus einem Fragetext und einer Menge an vorgegebenen Antwortmöglickeiten.
+Mehrere Antworten sind richtig.
+Es müssen alle richtigen Antworten markiert wereden. Es dürfen keine falschen Antworten markiert werden.
+
+### Lückentext
+
+Die Aufgabenstellung besteht aus einem Fragetext.
+Die Antwort ist ein kurzer Freitext.
+Es muss die richtige Antwort in den Freitext eingegeben werden.
+Die richtige Antwort kann ein konkreter Wert oder eine Eingabe passend zu einem regulären Ausdruck sein.
+
+### Ersetzen
+
+Aus einer Reihe aus Elementen (kann auch nur eins sein) muss ein Element durch ein oder mehrere andere Elemente ersetzt werden.
+Hierdurch wird die Reihe aus Elementen sowie deren Anzahl verändert.
+
+### Kombinieren
+
+Aus einer Reihe aus Elementen müssen mehrere Elemente (oder alle) zu einem anderen Element zusammengefasst werden.
+Hierdurch wird die Reihe aus Elementen sowie deren Anzahl verändert.
+
+### Objekte in der richtigen Reihenfolge auswählen
+
+Es muss in der richtigen Reihenfolge ein Element nach dem anderen aus einer Menge ausgewählt werden.
+Es müssen nicht alle Elemente von Anfang an bekannt sein.
+
+### Positionieren
+
+Positionierung bezeichnet im Allgemeinen die Aufgabe, Elemente aus einer Menge (z.B. Antwortmöglichkeiten)
+zu Positionen (bspw. Array-Index, Reihe-Spalte Kombination einer Matrix) zuzuordnen. Hierbei muss nicht jedes
+Element aus der Menge der Antwortmöglichkeiten auch einer Position zugeordnet werden. Allerdings muss jede Postion
+mit einem Element aus der Menge der Antwortmöglichkeiten belegt sein.
+
+## Spielmechanik
+
+### Ein GUI-Button drücken
+
+- Es wird eine GUI bestehend aus einem Feld für den Aufgabentext (Text oder Bild) und mehrere Buttons mit Antworten (Text oder Bild) angezeigt
+- Der Spieler muss auf den Button mit der Antwort klicken, die er abgeben möchte.
+
+### GUI Checkboxen anhaken
+
+- Es wird eine GUI bestehend aus einem Feld für den Aufgabentext (Text oder Bild) und mehrere Buttons mit Antworten (Text oder Bild) angezeigt
+- Der Spieler muss auf die Buttons mit der Antwort klicken, die er für richtig hält.
+- Mit einem Klick aktiviert er eine nicht aktive (nicht ausgewählte) Antwort oder deaktiviert eine aktivierte Antwort.
+- Mit einem Bestätigungsbutton muss der Spieler seine Antworten einloggen.
+
+### GUI Text eingeben
+
+- Es wird eine GUI bestehend aus einem Feld für den Aufgabentext (Text oder Bild) und einem Feld zur Texteingabe angezeigt.
+- Der Spieler muss seine Antwort in das Eingabefeld schreiben.
+- Mit einem Bestätigungsbutton muss der Spieler seine Antworten einloggen.
+
+
+### Item an NPC abgeben
+
+- Eine Menge an Items ist im Dungeon verteilt (dabei ist nicht relevant, wie und wo der Spieler die
+  Items findet und erhält)
+- Die Items können vom Spieler aufgesammelt werden
+- Bei Interaktion mit einem NPC kann der Spieler ein aufgesammeltes Item auswählen und an den NPC abgeben
```

Comment:
Ich verstehe nicht mehr den Unterschied zw. "Item an NPC" abgeben und "Item in Container ablegen". Das ist doch strukturell exakt identisch (siehe auch die identischen Beschreibungen)? Und warum darf ich dem NPC nur _ein_ Item geben, während ich einem Container gleich mehrere Items (nacheinander) geben darf?